tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -155,6 +155,14 @@ export class WebGPUBackend extends KernelBackend {
         type: 'timestamp',
         count: 2,
       });
+    } else {
+      console.warn(
+          `This device doesn't support timestamp-query extension. ` +
```

Comment:
nit: remove the intermediate `+`. Only one pair of `` is enough.

Reply:
Only one pair of `` will separate long warnings to several lines and keep extra spaces at the beginning of each line. ![image](https://user-images.githubusercontent.com/7088194/181490108-f8f4adea-47a8-4ef7-bda3-1207eb2b1929.png)
Below is what is looks like now (in one line):
![image](https://user-images.githubusercontent.com/7088194/181490728-8a925372-f735-4e0b-aa1f-67627366747f.png)

Reply:
I think you can adjust your output layout.
For example:
```
      console.warn(`
This device doesn't support timestamp-query extension.
Start Chrome browser with flag
--disable-dawn-features=disallow_unsafe_apis then try again.
Or zero will shown for the kernel time when profiling mode is
enabled. Using performance.now is not workable for webgpu since
it doesn't support synchronously to read data from GPU.`);

Reply:
I thought this also break one sentence into 6 lines, sometimes may cause the typography issue.
For example when console window is less than 70 width(debugging in a small corner), 6 lines will became 6 paragraphs(12+ lines), instead 1 line message will stay in 1 paragraph, easy to read.